docs/RelayAPI.md

```diff
@@ -1089,6 +1089,33 @@ curl --header "Content-Type: application/json" \
 
 ---
 
+### Status of meta transaction
+Get the status of a meta transaction for an identity.
+
+#### Request
+```
+GET /identities/:identityAddress/meta-transactions-status/:metaTransactionHash
```

Comment:
should we not maybe have `meta-transaction` and than one field is status, so we can just add other fields later, like when it was included and so on.